src/vscode-bicep/src/commands/deploy.ts

```diff
@@ -428,7 +429,8 @@ export class DeployCommand implements Command {
       // open it in vscode.
       if (
         parametersFileUpdateOption !== ParametersFileUpdateOption.None &&
-        parametersFileName !== null
+        parametersFileName !== undefined &&
```

Comment:
For future reference: It's actually better to just say:

if (parametersFileUpdateOption !== ParametersFileUpdateOption.None && parametersFileName && parametersFilePath)

This does a test for "truthy" values in ts/js, meaning it's any non-null and non-undefined value (for strings at least).